### DISCUSSION TITLE
How can I access to radio with tab?

### DISCUSSION THREAD

**Discussion by soymi1k** [rsuite/rsuite#3133] [Q&A] [answered]
I want to access Radio with tab but
The component prevented to access with keyboard
how can I access?

I have tried tabIndex prop

https://user-images.githubusercontent.com/73974978/227450620-a2a95bcb-9e8a-4a82-9069-a2f23d94a551.mov

**Comment by SevenOutman** [comment, marked as answer]
RadioGroup with `appearance="picker"` lacks support for keyboard controls for the moment. This should be resolved in following releases.
